hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -130,10 +132,10 @@
    * |----------------------------------------------------------------------|
    * | transactionInfoTable| #TRANSACTIONINFO -> OMTransactionInfo          |
    * |----------------------------------------------------------------------|
-   * | tenantUserTable    |  tenant user name -> a tenant (organization)    |
-   * | tenantStateTable   |  tenant user name -> OmDBTenantInfo             |
-   * | tenantGroupTable   |  tenant user name -> [tenant group A, B, ...]   |
-   * | tenantRoleTable    |  tenant user name -> roles [admin, roleB, ...]  |
+   * | tenantAccessIdTable|  accessId -> OmTenantAccessIdInfo               |
+   * | tenantStateTable   |  accessId -> OmDBTenantInfo                     |
+   * | tenantGroupTable   |  accessId -> [tenant group A, B, ...]           |
+   * | tenantRoleTable    |  accessId -> roles [admin, roleB, ...]          |
```

Comment:
Nit. New section in the comment block for multi tenant tables, with a line between each table to match formatting in the rest of this block?